ui/src/App/index.jsx

```diff
@@ -30,18 +31,14 @@ import introspectionQueryResultData from '../fragments/fragmentTypes.json';
 import { route } from '../utils/prop-types';
 import AuthController from '../auth/AuthController';
 
-@hot(module)
-export default class App extends Component {
-  static propTypes = {
-    routes: arrayOf(route).isRequired,
-  };
-
-  fragmentMatcher = new IntrospectionFragmentMatcher({
+const AppHooks = ({ routes }) => {
```

Comment:
We can just name it `App`.

Reply:
Apologies for the delayed reply. I will rebase on master soon.

Reply:
I have rebased it on master, had to resolve some conflicts and it runs on local env. Also, eslint still has issues with webpack alias for 'react-dom' . 
Please have a review. Thanks.

Reply:
Hi, any update on this?